plugins/chat/app/services/chat/update_message.rb

```diff
@@ -134,7 +138,8 @@ def should_create_revision(new_message, prev_message, guardian)
     def publish(message:, guardian:, contract:, **)
       edit_timestamp = context.revision&.created_at&.iso8601(6) || Time.zone.now.iso8601(6)
 
-      ::Chat::Publisher.publish_edit!(message.chat_channel, message)
+      ::Chat::Publisher.publish_edit!(message.chat_channel, message, streaming: contract.streaming)
```

Comment:
Same as before but just need to decide if updating the message requires us to set streaming as `true` at any point